src/demos/calcite-pick-list-advanced.html

```diff
@@ -0,0 +1,451 @@
+              <calcite-action slot="secondaryAction" id="action-test" label="click-me" onClick="console.log('clicked');">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
+                  <path
+                    d="M7.5 0A7.5 7.5 0 1 0 15 7.5 7.509 7.509 0 0 0 7.5 0zm.001 14.1A6.6 6.6 0 1 1 14.1 7.5a6.608 6.608 0 0 1-6.599 6.6zM7.5 5.5a1 1 0 1 1 1-1 1.002 1.002 0 0 1-1 1zM7 7h1v5H7zm2 5H6v-1h3z"
+                  />
+                </svg>
+              </calcite-action>
+            </calcite-pick-list-item>
+          </calcite-pick-list-group>
+          <calcite-pick-list-group text-group-title="colors">
+            <calcite-pick-list-item
+              text-label="2018 Total Housing Units (Esri)"
+              text-description="TOTHU_CY"
+              value="TOTHU_CY"
+            >
+              <calcite-action slot="secondaryAction" id="action-test" label="click-me" onClick="console.log('clicked');">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
+                  <path
+                    d="M7.5 0A7.5 7.5 0 1 0 15 7.5 7.509 7.509 0 0 0 7.5 0zm.001 14.1A6.6 6.6 0 1 1 14.1 7.5a6.608 6.608 0 0 1-6.599 6.6zM7.5 5.5a1 1 0 1 1 1-1 1.002 1.002 0 0 1-1 1zM7 7h1v5H7zm2 5H6v-1h3z"
+                  />
+                </svg>
+              </calcite-action>
+            </calcite-pick-list-item>
+            <calcite-pick-list-item
+              text-label="2018 Owner Occupied Housing Units (Esri)"
+              text-description="OWNER_CY"
+              value="OWNER_CY"
+            >
+              <calcite-action slot="secondaryAction" id="action-test" label="click-me" onClick="console.log('clicked');">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
+                  <path
+                    d="M7.5 0A7.5 7.5 0 1 0 15 7.5 7.509 7.509 0 0 0 7.5 0zm.001 14.1A6.6 6.6 0 1 1 14.1 7.5a6.608 6.608 0 0 1-6.599 6.6zM7.5 5.5a1 1 0 1 1 1-1 1.002 1.002 0 0 1-1 1zM7 7h1v5H7zm2 5H6v-1h3z"
+                  />
+                </svg>
+              </calcite-action>
+            </calcite-pick-list-item>
+          </calcite-pick-list-group>
+        </calcite-pick-list>
+        <script>
+          const pickList6 = document.querySelector("#three_rtl");
+          pickList6.addEventListener("calcitePickListSelectionChange", (event) => {
+            console.log(event.detail);
+          });
+        </script>
+      </section>
+      <section class="example-container" style="width: 28vw; margin-top: 2em;">
+          <h2>Maquette Version</h2>
+          <div id="app"></div>
+      </section>
+    </main>
+    <script>
+      function toggleProperty(property) {
+        const blocks = document.getElementsByTagName("calcite-pick-list");
+        for (i = 0; i < blocks.length; i++) {
+          blocks[i].toggleAttribute(property);
+        }
+      }
+    </script>
+    <script>
+      function renderMaquette() {
```

Comment:
I don't think we should include this in our demos. At least from reading this page, there's no context on why this exists, doesn't look exhaustive, and also opens a can of worms whether we need to update other demos for consistency. This is testing component creation via properties through maquette, which we could cover in E2E testing. Maybe we could use maquette as a util to quickly set up these tests since it does make it simple to set up, but I see this as something separate and test-related.

Reply:
Agreed. We shouldn't be testing integrations in this repo. I missed this